File: Day_4/Q13.java
import java.util.*;
class Q13{
    //Generating nth Fibonacci Number
    static int fib(int n){
        if(n <= 1) return n;
        return fib(n-1) + fib(n-2);
    }
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        if(a < 0 || b < 0 || a > b){
            System.out.println("Invalid Input");
        }
        //Looping And Printing Fibonacci Number For All i ∈ [a,b]
        else{
                for (int i = a; i <= b; i++) {
                    System.out.print(fib(i) + "\t");   
                }
        }
        sc.close();
    }
}

File: Day_4/Q13.cpp
#include <iostream>
using namespace std;
//Generating nth Fibonacci Number
int fib(int n){
    if(n <= 1) return(n);
    return fib(n-1) + fib(n-2); 
}
int main(){
    int a,b;
    cin >> a >> b;
    if(a < 0 || b < 0 || a > b){
        cout << "Invalid Input";
        return 0;
    }
    //Looping And Printing Fibonacci Number For All i ∈ [a,b]
    for(int i = a;i<=b;i++){
        cout << fib(i) << "\t";
    }
    return 0;
}